### PR TITLE
Test to verify servicelog.

### PR DESCRIPTION
**Pull request by Pavithra1602**
The servicelog packages provides a library for logging service-related
events to the servicelog database, and a number of command-line
utilities for viewing the contents of the database.

Signed-off-by: pavrampu@linux.vnet.ibm.com <pavrampu@in.ibm.com>

### CODE REVIEW

File: generic/servicelog.py [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+            self.run_cmd("servicelog --id=1")
+            self.run_cmd("servicelog --query='severity>=$WARNING AND closed=0'")
+            self.run_cmd("servicelog --serviceable=yes")
+            self.run_cmd("servicelog --serviceable=no")
+            self.run_cmd("servicelog --serviceable=all")
+            self.run_cmd("servicelog --query='serviceable=1 AND closed=0'")
+            self.run_cmd("servicelog --repair_action=yes")
+            self.run_cmd("servicelog --repair_action=no")
+            self.run_cmd("servicelog --repair_action=all")
+            self.run_cmd("servicelog --repair_action=yes")
```
Comment (balamuruhans) [inline generic/servicelog.py]: we have `self.run_cmd("servicelog --repair_action=yes")` 2 times, should we have run twice, also can we have a list of cmds and use for loop,
```
service_log_cmds = ["--help", "--version", ..]
for cmd in service_log_cmds:
    self.run_cmd("servicelog %s" % cmd)
```

File: generic/servicelog.py [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+                          "RTAS Event")
+        self.log.info("=====3 -  Cleaning events from the servicelog "
+                      "database =======")
+        self.run_cmd("servicelog_manage --truncate events --force")
+        NoofRecords = 0
+        self.log.info("======4 - Checking if the events database was"
+                      " really cleaned up=========")
+        NoofRecords = process.system_output("servicelog --dump |"
+                                            " grep \"Power Platform (RTAS) Event\" | wc -l", shell=True)
+        if int(NoofRecords) != 0:
```
Comment (PraveenPenguin) [inline generic/servicelog.py]: can this be like
```
if int(NoofRecords):
   self.is_fail += 1
```

File: generic/servicelog.py.data/rtas_events/v3_fan_failure [hunk 3]
```diff
@@ -0,0 +1,6 @@
+RTAS: 1001 -------- RTAS event begin --------
+RTAS 0: 0354004000000028c600950015071800
+RTAS 1: 2003072200000001a800000000002329
+RTAS 2: 000000010000001e0000000900000000
+RTAS: 1001 -------- RTAS event end ----------
+
```
Comment (PraveenPenguin) [inline generic/servicelog.py.data/rtas_events/v3_fan_failure]: can this white will be removed ?

File: generic/servicelog.py.data/rtas_events/v3_predictive_cpu_failure [hunk 4]
```diff
@@ -0,0 +1,6 @@
+RTAS: 1 -------- RTAS event begin --------               
+RTAS 0: 03441107 00000028 9E009F00 15071800
+RTAS 1: 20030722 10060004 a0000000 00002329
+RTAS 2: 00000001 0000001e 00000009 00000000
+RTAS: 1 -------- RTAS event end ----------   
+
```
Comment (PraveenPenguin) [inline generic/servicelog.py.data/rtas_events/v3_predictive_cpu_failure]: here as well

File: generic/servicelog.py.data/rtas_events/v4_io_bus_failure [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+RTAS 11: 1711005600000000000c464617110037
+RTAS 12: c0c00100000c46461711005700000000
+RTAS 13: 000c46460407013a14000000000c4646
+RTAS 14: 0407012b000000000010464604070168
+RTAS 15: 0000000000000000000c464604070134
+RTAS 16: 40001100000c46460407015400000000
+RTAS 17: 000c46460407013500641100000c4646
+RTAS 18: 04070155000000000002
+RTAS: 1 -------- RTAS event end ----------
+
```
Comment (PraveenPenguin) [inline generic/servicelog.py.data/rtas_events/v4_io_bus_failure]: here also

**Comment by Pavithra1602** [comment]
Raised new PR.

**Pull request closed without merging**